<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPhaseTiming</class>
 <widget class="QDialog" name="DialogPhaseTiming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>阶段配时</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>186</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>阶段绿灯时间</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinblue">
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>阶段黄灯时间</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinyellow">
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>阶段红灯时间</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinred">
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="butquery">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询
已设</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>60</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>671</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>放行相位</string>
   </property>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QCheckBox" name="phase1">
      <property name="text">
       <string>相位1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase2">
      <property name="text">
       <string>相位2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase3">
      <property name="text">
       <string>相位3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase4">
      <property name="text">
       <string>相位4</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase5">
      <property name="text">
       <string>相位5</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase6">
      <property name="text">
       <string>相位6</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase7">
      <property name="text">
       <string>相位7</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase8">
      <property name="text">
       <string>相位8</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QCheckBox" name="phase9">
      <property name="text">
       <string>相位9</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase10">
      <property name="text">
       <string>相位10</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase11">
      <property name="text">
       <string>相位11</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase12">
      <property name="text">
       <string>相位12</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase13">
      <property name="text">
       <string>相位13</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase14">
      <property name="text">
       <string>相位14</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase15">
      <property name="text">
       <string>相位15</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phase16">
      <property name="text">
       <string>相位16</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkisparm">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>246</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>是否优选公共周期</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinnum">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>296</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>已设行号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>29</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>阶段号</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxplannum">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="hang">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>299</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>阶段选项参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>29</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>阶段配时方案号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butclearall">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清除
全表</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>273</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="butfind">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butok">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butclear">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
